src/programs.rkt

```diff
@@ -351,3 +360,51 @@
        (and (operator-info op field) (andmap loop args))]
       [(? variable?) true]
       [(? constant?) (or (not (symbol? expr)) (constant-info expr field))])))
+
+;; Conversions to and from (<= 1.4) and "parameterized" Herbie
+;; Needed for src/core/taylor.rkt, src/core/reduce.rkt, src/syntax/rules.rkt
+;; Similar to de/resugaring but not quite
```

Comment:
What's the difference? Is it just casts turning into annotations? It'd be really nice not to have another IL.

Reply:
The only difference is how it handles `neg` and posit type conversions. I agree though. It's annoying to have another set. However, Herbie does have 2 intermediary formats now.